src/components/checkbox/Checkbox.base.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+
+import Shape from '../../foundations/shape';
+import Grid from '../../foundations/layout';
+import Icon, { TIconSize } from '../icon';
+import Text, { TTextSizeToken } from '../text';
+import { Utils } from '../../shared';
+
+import PCheckbox from './Checkbox.props';
+import { DEFAULT_CHECKBOX_SIZE } from './Checkbox.constants';
+
+const CheckboxBase: React.FC<PCheckbox> = ({
+    label = 'Public',
+    size = DEFAULT_CHECKBOX_SIZE,
```

Comment:
default values should be defined in `Checkbox.tsx`

Reply:
I need to define it here in order to use it with as an index type with 'CHECKBOX_SIZE_MAPPING'

Reply:
mmh, i will check today how it influences extraction of args in the `ArgsTable`